test/jdk/lib/client/ExtendedRobot.java

```diff
@@ -106,180 +103,8 @@ public ExtendedRobot(GraphicsDevice screen) throws AWTException {
-        super.waitForIdle();
-        delay(delayValue);
-    }
-
-    /**
-     * Waits until all events currently on the event queue have been processed with delay
-     * {@link #getSyncDelay()} after syncing threads. It uses more advanced method of
-     * synchronizing threads unlike {@link java.awt.Robot#waitForIdle()}
-     *
-     * @throws  sun.awt.SunToolkit.IllegalThreadException if called on the AWT event
-     *          dispatching thread
-     *
-     * @see     #waitForIdle(int)
-     */
-    @Override
-    public synchronized void waitForIdle() {
-        waitForIdle(syncDelay);
-    }
-
-    /**
-     * Move the mouse in multiple steps from where it is
-     * now to the destination coordinates.
-     *
-     * @param   x   Destination point x coordinate
-     * @param   y   Destination point y coordinate
-     *
-     * @see     #glide(int, int, int, int)
-     */
-    public void glide(int x, int y) {
-        Point p = MouseInfo.getPointerInfo().getLocation();
-        glide(p.x, p.y, x, y);
-    }
-
-    /**
-     * Move the mouse in multiple steps from where it is
-     * now to the destination point.
-     *
-     * @param   dest    Destination point
-     *
-     * @see     #glide(int, int)
-     */
-    public void glide(Point dest) {
-        glide(dest.x, dest.y);
-    }
-
-    /**
-     * Move the mouse in multiple steps from source coordinates
-     * to the destination coordinates.
-     *
-     * @param   fromX   Source point x coordinate
-     * @param   fromY   Source point y coordinate
-     * @param   toX     Destination point x coordinate
-     * @param   toY     Destination point y coordinate
-     *
-     * @see     #glide(int, int, int, int, int, int)
-     */
-    public void glide(int fromX, int fromY, int toX, int toY) {
-        glide(fromX, fromY, toX, toY, DEFAULT_STEP_LENGTH, DEFAULT_SPEED);
-    }
-
-    /**
-     * Move the mouse in multiple steps from source point to the
-     * destination point with default speed and step length.
-     *
-     * @param   src     Source point
-     * @param   dest    Destination point
-     *
-     * @see     #glide(int, int, int, int, int, int)
-     */
-    public void glide(Point src, Point dest) {
-        glide(src.x, src.y, dest.x, dest.y, DEFAULT_STEP_LENGTH, DEFAULT_SPEED);
-    }
-
-    /**
-     * Move the mouse in multiple steps from source point to the
-     * destination point with given speed and step length.
-     *
-     * @param   srcX        Source point x cordinate
-     * @param   srcY        Source point y cordinate
-     * @param   destX       Destination point x cordinate
-     * @param   destY       Destination point y cordinate
-     * @param   stepLength  Approximate length of one step
-     * @param   speed       Delay between steps.
-     *
-     * @see     #mouseMove(int, int)
-     * @see     #delay(int)
-     */
-     public void glide(int srcX, int srcY, int destX, int destY, int stepLength, int speed) {
```

Comment:
I hadn't noticed that this over-ride is both removed here and not added to Robot.
If this were available, then perhaps those setters and getters for the current values aren't needed !
What was the reason for not including it ? No one ever used it or something else ?